### PR TITLE
Adding config scripts so we can run automated with `-s` flag

### LINKED ISSUE #95: Add namespace parameter to k8s scripts
Allow users to specify KUBERNETES_NAMESPACE which should default to the `default` namespace

### LINKED ISSUE #96: Add scripts that configure environment when not running setup
Currently the setup step sets env vars and determines the SERVER_URI. We should add a config step that can set the same env vars when we don't run the setup.

We should be able to run this:

```
./run.sh -p <platform> -t -c
./run.sh -p <platform> -s
```

without having to explicitly set any env vars

### PR DESCRIPTION
**Pull request by trisberg**
- we should still see the environment set properly for the server and binder settings

Resolves #96 
Resolves #95

### CODE REVIEW

**Comment by trisberg** [comment]
Rebased

File: kubernetes/server/create.sh [hunk 1]
```diff
@@ -1,24 +1,20 @@
 #!/bin/bash
 
 function kubectl_create() {
-  kubectl create -f scdf.yml
+  kubectl create -f scdf.yml --namespace $KUBERNETES_NAMESPACE
   READY_FOR_TESTS=1
   for i in $( seq 1 "${RETRIES}" ); do
-    SERVER_URI=$(kubectl get svc scdf | grep scdf | awk '{print $3}')
+    SERVER_URI=$(kubectl get svc scdf --namespace $KUBERNETES_NAMESPACE | grep scdf | awk '{print $3}')
```
Comment (ilayaperumalg) [inline kubernetes/server/create.sh]: Should this be `awk '{print $2}'`?

as I see the result for instance:

```
scdf      10.75.249.66   35.184.121.164   80/TCP    3m
```

Comment (ilayaperumalg) [inline kubernetes/server/create.sh]: nevermind. I think the current one is correct.

File: kubernetes/server/create.sh [hunk 2]
```diff
@@ -1,24 +1,20 @@
...
     [ '<pending>' != $SERVER_URI ] && READY_FOR_TESTS=0 && break
```
Comment (ilayaperumalg) [inline kubernetes/server/create.sh]: Instead of checking for `pending`, I guess we need to check for `80/TCP` as that will be the result for `awk '{print $3}')` until the external IP is assigned

Comment (ilayaperumalg) [inline kubernetes/server/create.sh]: @trisberg @viniciusccarvalho What do you think about these changes to better handle netcat_port check: https://github.com/ilayaperumalg/spring-cloud-dataflow-acceptance-tests/commit/65cebd60ba83791580b2e7c2bf6b3dfbed0bbd79

I ran into issues when running the tests against `kubernetes` and with these changes I could run fine.

Comment (viniciusccarvalho) [inline kubernetes/server/create.sh]: Hi @ilayaperumalg, it's weird because the netcat port is used to check local ports with nc, if you remove that then it will probably fail for local tests. I did not know that k8s was using it as well, but I think that your change may break local, have you tried local too?

Comment (trisberg) [inline kubernetes/server/create.sh]: Do you have an older `kubectl` version, the newer ones say <pending> when the external IP hasn't been assigned yet

Comment (trisberg) [inline kubernetes/server/create.sh]: Also, why did you add ` --validate=false`?

Comment (ilayaperumalg) [inline kubernetes/server/create.sh]: It was the `kubectl` version which had issues. After upgrading to the latest, I could start the kubernetes tests fine.

The earlier version had `validation` errors against the YAML files and also wasn't returning `pending`. Thanks @trisberg @viniciusccarvalho
